smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -85,21 +92,21 @@ function displayReaderList(readers) {
     goog.asserts.assert(readersListElement);
 
     var indicatorClasses = 'reader-state-indicator reader-state-indicator-' +
-                           reader['status'];
+                            reader['status'];
```

Comment:
nit: There's still some unwanted indentation change in this line. I don't think the code in master has an error here - the `reader` word should be indented with the `'` character, since it's the beginning of the expression.

Reply:
oh ok i thought it should align with the beginning of the word